plugins/woocommerce-blocks/assets/js/blocks/product-elements/add-to-cart-form/edit.tsx

```diff
@@ -17,15 +17,32 @@ import { isBoolean } from '@woocommerce/types';
  */
 import './editor.scss';
 import { useIsDescendentOfSingleProductBlock } from '../../../atomic/blocks/product-elements/shared/use-is-descendent-of-single-product-block';
-import { QuantitySelectorStyle, Settings } from './settings';
+import { QuantitySelectorStyle, AddToCartSettings } from './settings';
 
 export interface Attributes {
 	className?: string;
 	isDescendentOfSingleProductBlock: boolean;
 	quantitySelectorStyle: QuantitySelectorStyle;
 }
 
-const Edit = ( props: BlockEditProps< Attributes > ) => {
+export type FeaturesKeys =
+	| 'isStepperLayoutFeatureEnabled'
+	| 'isBlockifyAddToCartEnabled';
+
+export type FeaturesProps = {
+	[ key in FeaturesKeys ]?: boolean;
+};
+
+export type UpdateFeaturesType = ( key: FeaturesKeys, value: boolean ) => void;
+
+// Pick the value of the "blockify add to cart flag"
+const isBlockifyAddToCartEnabled = getSettingWithCoercion(
+	'isBlockifyAddToCartEnabled',
+	false,
+	isBoolean
+);
```

Comment:
I guess this is temporary, but wondering if we could define `isStepperLayoutFeatureEnabled` and `isBlockifyAddToCartEnabled` next to each other?

Reply:
yes, we should indeed. The thing is, I didn't want to change something that seems to be out of the scope of this PR. Maybe a follow-up?